### ISSUE TITLE
Beardify — Lemoji beard structure text generator

### ISSUE THREAD

**Issue by gustavwilliam** [python-discord/seasonalbot#436]
## Description

A command that would let the user easily convert text to a beard structure, using the beard structure lemojis.

Here's `PyDis` using the beard structure, in a Discord channel!

![image](https://user-images.githubusercontent.com/65498475/92949485-71670680-f45b-11ea-869b-53570313865c.png)

## Reasoning
Being able to write with lemoji beard, without having to reinvent the characters every time, would be fantastic.

## Proposed Implementation

A command like `.beardify <text>` could be used. SeasonalBot would return the text, written using the beard structure lemojis. Since the beard lemojis are available in the server, it would be easy to copy+paste the text yourself. If it gets too wide to fit, the beardified text could be returned as a png instead.

Only alpha numerical or ASCII would be allowed as a start.

## Would you like to implement this yourself?

- [ ] I'd like to implement this feature myself
- [x] Anyone can implement this feature

**Comment by gustavwilliam** [comment]
TL;DR: I've begun implementing this myself.

I have a few updates to the implementation of it:
- Resulting text will be returned as a png
- Emojis will be on a separate server
- Multiple different fonts can be used (one exists as of now). The font height between fronts and letter width within fonts is variable

I'll write more about the custom emoji font format soon, if anyone is interested in contributing with more fonts!

**Comment by Xithrius** [comment]
I've communicated with @gustavwilliam. This is still in the works, and is a bit on hold. 

I hope this feature works out, very cool stuff.

**Comment by Xithrius** [comment]
@gustavwilliam How's this feature going?
